NDLANO/ndla-frontend
# Implement new click toggle

To test: 
- run local article converter from branch "Figure-component-fixes"
- Check that all clickToggles work (in authorByLine, FigureLicense etc)
